packages/web3/src/web3.ts

```diff
@@ -31,7 +31,7 @@ import { initAccountsForContext } from './accounts';
 import { Web3EthInterface } from './types';
 import { Web3PkgInfo } from './version';
 
-export default class Web3 extends Web3Context<EthExecutionAPI> {
```

Comment:
This will break minified web3.js build https://github.com/ChainSafe/web3.js/issues/5345

Reply:
[This](https://github.com/ChainSafe/web3.js/pull/5393/commits/755d6d65fb490493b3fbf529589f27069e6e7f59) commit updates `Web3` to be exported via `default` and named. I've created a [test example](https://github.com/spacesailor24/web3-5393-min-test) that uses the `web3.min.js` and calls various functions to ensure the minified build works